se.cendio.tlclient.json

```diff
@@ -0,0 +1,58 @@
+{
+	"app-id": "se.cendio.tlclient",
+	"runtime": "org.freedesktop.Platform",
+	"runtime-version": "21.08",
+	"sdk": "org.freedesktop.Sdk",
+	"command": "tlclient-openconf",
+	"tags": [
+		"proprietary"
+	],
+	"finish-args": [
+		"--persist=.fltk",
+		"--persist=.thinlinc",
+		"--share=ipc",
+		"--share=network",
+		"--socket=x11",
+		"--socket=pulseaudio"
```

Comment:
File system access is also needed for local drives to work. And for public key authentication.

Can this be interactive? So the user can choose how much access they want to give?

Reply:
> Can this be interactive? So the user can choose how much access they want to give?

Sort of. There is [Flatseal](https://github.com/tchx84/Flatseal) which is a GUI application for configuring this, but the proper way is to use desktop portals for filesystem access. See https://github.com/flatpak/xdg-desktop-portal. Unless you want to go about integrating that (I'm assuming FLTK does not provide support for this for you), it would probably be easiest to just always allow access to these directories to not confuse users. What kind of access is necessary?

Reply:
For local drives constant file access is needed, which I don't think there is a portal for.

I'd say the user's home drive is a minimum. Some user want to export things outside of that though, so complete filesystem access would be needed for them.

Reply:
I added host filesystem access.

Reply:
We can confirm this works in our testing.